stories/carousel/carousel.stories.js

```diff
@@ -477,3 +476,133 @@ carouselNoNavigation.parameters = {
+                    role="img"
+                    aria-label="Carousel Image 6">
+                </div>
+            </div>
+
+            <div class="fd-carousel__item fd-carousel__item--active">
+                <div
+                    style="
+                        width:100%;
+                        background-image: url(assets/images/backgrounds/Godafoss_waterfall_in_northern_Iceland.jpg);
+                        height: 15.5rem;
+                        background-size:cover;"
+                    role="img"
+                    aria-label="Carousel Image 7">
+                </div>
+            </div>
+        </div>
+        <div class="fd-carousel__page-indicator-container">
+            <button
+                class="fd-button fd-carousel__button fd-carousel__button--left"
+                data-slide="prev"
+                aria-label="Go to previous item">
+                <i class="sap-icon--slim-arrow-left"></i>
+            </button>
+            <ol class="fd-carousel__page-indicators">
+                <li
+                    data-slide-to="1"
+                    class="fd-carousel__page-indicator"></li>
+                <li
+                    data-slide-to="2"
+                    aria-label="Displaying item 2 of 3"
+                    class="fd-carousel__page-indicator fd-carousel__page-indicator--active"></li>
+                <li
+                    data-slide-to="3"
+                    class="fd-carousel__page-indicator"></li>
+            </ol>
+            <button
+                class="fd-button fd-carousel__button fd-carousel__button--right"
+                data-slide="next"
+                aria-label="Go to next item">
+                <i class="sap-icon--slim-arrow-right"></i>
+            </button>
+        </div>
+    </div>
+    <div style="display: none;" role="region" id="carousel-11" aria-live="polite">
+        Displaying item 2 of 3
+    </div>
+</div>
+`;
+
+horizontalCarousel.storyName = 'Items in horizontal direction';
+horizontalCarousel.parameters = {
+    docs: {
+        iframeHeight: 900,
+        storyDescription: `Carousel will have items which will spread horizontally and will be visible on navigation. For translation to work, all items should be in DOM.
+`
+    }
+};
+
+export const error = () => `<div style="display: flex; flex-direction: column; align-items: center; background: #CCD1D1;">
+    <h4>Error in loading items</h4>
+    <div
+        class="fd-carousel fd-carousel--no-navigation"
+        data-ride="carousel"
+        style="margin-bottom: 3rem; max-width: 60rem;">
+        <div class="fd-carousel__content fd-carousel__content--error-message" style="text-align: center; padding: 1rem; min-height: 15.5rem;">
+            <div class="docs-column-flex">
+                <div class="fddocs-horizontal-center">
+                    <i class="sap-icon--document"></i>
+                </div>
+                Items could not be loaded
+            </div>
```

Comment:
This part will not be consumed in ngx and people will have to add custom css there

Reply:
It was decided during tech spec review that Error message will be coming in Ngx as <ng-content>.
so only placing of error message is take care.

formatting of error message like having Icon and Error message vertically and icon horizontally centred will have to be taken care by application.

Do you want to have that styling present here?
if so, the during application development, those classes will be exposed.

let me know your thought and correct me.

Reply:
Hi @DeepakSap14 the specs clearly mention that the Empty State (when no pages are loaded) should use the Message Page component. It's present in both fund-styles and fund-ngx. You don't have to use custom css, just use the component.
<img width="679" alt="Screen Shot 2021-06-16 at 8 52 44 AM" src="https://user-images.githubusercontent.com/39598672/122223029-cf0fff80-ce80-11eb-996c-5fa61ae98b1c.png">